ext/standard/basic_functions.stub.php

```diff
@@ -79,6 +79,161 @@ function sort(array &$arg, int $sort_flags = SORT_REGULAR): bool {}
 
 function rsort(array &$arg, int $sort_flags = SORT_REGULAR): bool {}
 
+function usort(array &$arg, callable $cmp_function): bool {}
+
+function uasort(array &$arg, callable $cmp_function): bool {}
+
+function uksort(array &$arg, callable $cmp_function): bool {}
+
+function end(array &$arg) {}
+
+function prev(array &$arg) {}
+
+function next(array &$arg) {}
+
+function reset(array &$arg) {}
+
+function current(array $arg) {}
+
+/** @return int|string|null */
+function key(array $arg) {}
+
+function min(...$args) {}
```

Comment:
This would technically allow zero arguments as well ... I'm wondering if we should be using a `min($arg, ...$args)` signature to show that at least one arg is required?

Reply:
Wouldn't that be a BC break from the old arginfo?

Reply:
As this is not a method, there isn't any BC we can break.